src/components/PopupWithForm.js

```diff
@@ -5,18 +5,23 @@ export class PopupWithForm extends Popup {
     this._popUpForm = this._popUpElement.querySelector(".modal__form");
     this._handleModalSubmit = handleModalSubmit;
   }
-  closeModal() {
-    this._popUpForm.reset();
-    super.closeModal();
-  }
+  // closeModal() {
+  //   this._popUpForm.reset();
+  //   super.closeModal();
+  // }
   setEventListeners() {
     super.setEventListeners();
     this._popUpForm.addEventListener("submit", (e) => {
-      e.preventDefault();
       const modalInputs = this._getInputValues();
-      this._handleModalSubmit(modalInputs);
+      e.preventDefault();
+      this._popUpForm.querySelector(".modal__form-button").innerText =
```

Comment:
`this._popUpForm.querySelector(".modal__form-button")`

**Could be improved**

All elements that are found every time you call a method should be defined as **class fields** in the constructor, `_setEventListeners` (or in `generateCard`) only **1 time** not to waste resources on searching them again and again.

`Class fields` (the so called **class variables** with `this`) are accessible inside any method of the class. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.
There can be `this._inputs`, `this._submitButton`, `this._cardImage`, `this._form` , `this._likeButton` and so on.